<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-KBKTO19U" failures="3" tests="8" name="runners.TestRunner" time="184.770" errors="1" timestamp="2025-09-06T21:56:50 IST" skipped="0">
  <testcase classname="runners.TestRunner" name="runScenario" time="12.815"/>
  <testcase classname="runners.TestRunner" name="runScenario" time="37.955">
    <failure message="Empty password message not found! expected object to not be null" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Empty password message not found! expected object to not be null
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.assertNotNull(Assert.java:1496)
at stepdefinitions.LoginSteps.emptyPasswordMessageShouldBeShown(LoginSteps.java:68)
at ✽.an error message for empty password should be shown(file:///C:/Users/nathj/eclipse-workspace/bstackdemo/src/test/resources/features/login.feature:38)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runners.TestRunner" name="runScenario" time="35.357"/>
  <testcase classname="runners.TestRunner" name="runScenario" time="11.603"/>
  <testcase classname="runners.TestRunner" name="runScenario" time="16.170">
    <failure message="Locked user message not found! expected object to not be null" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Locked user message not found! expected object to not be null
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.assertNotNull(Assert.java:1496)
at stepdefinitions.LoginSteps.lockedUserMessageShouldBeShown(LoginSteps.java:43)
at ✽.a locked user message should be shown(file:///C:/Users/nathj/eclipse-workspace/bstackdemo/src/test/resources/features/login.feature:20)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runners.TestRunner" name="runScenario" time="36.629">
    <error message="Expected condition failed: waiting for element to be clickable: By.xpath: //div[contains(@id,&#039;react-select&#039;) and text()=&#039;invaliduser&#039;] (tried for 5 second(s) with 500 milliseconds interval)
Build info: version: &#039;4.35.0&#039;, revision: &#039;1c58e5028b&#039;
System info: os.name: &#039;Windows 11&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;21.0.7&#039;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 139.0.7258.155, chrome: {chromedriverVersion: 139.0.7258.154 (9e0d6b2b47f..., userDataDir: C:\Users\nathj\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:55563}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:55563/devtoo..., se:cdpVersion: 139.0.7258.155, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: f589c936dcdfcdd38d2482cc6f973678" type="org.openqa.selenium.TimeoutException">
      <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for element to be clickable: By.xpath: //div[contains(@id,'react-select') and text()='invaliduser'] (tried for 5 second(s) with 500 milliseconds interval)
Build info: version: '4.35.0', revision: '1c58e5028b'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.7'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 139.0.7258.155, chrome: {chromedriverVersion: 139.0.7258.154 (9e0d6b2b47f..., userDataDir: C:\Users\nathj\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:55563}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:55563/devtoo..., se:cdpVersion: 139.0.7258.155, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: f589c936dcdfcdd38d2482cc6f973678
at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:84)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:228)
at pages.LoginPage.enterUsername(LoginPage.java:35)
at stepdefinitions.LoginSteps.userLogsIn(LoginSteps.java:22)
at ✽.the user logs in with username "invaliduser" and password "wrongpassword"(file:///C:/Users/nathj/eclipse-workspace/bstackdemo/src/test/resources/features/login.feature:25)
]]>
    </error>
  </testcase> <!-- runScenario -->
  <testcase classname="runners.TestRunner" name="runScenario" time="14.375"/>
  <testcase classname="runners.TestRunner" name="runScenario" time="19.866">
    <failure message="Empty username message not found! expected object to not be null" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Empty username message not found! expected object to not be null
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.assertNotNull(Assert.java:1496)
at stepdefinitions.LoginSteps.emptyUsernameMessageShouldBeShown(LoginSteps.java:61)
at ✽.an error message for empty username should be shown(file:///C:/Users/nathj/eclipse-workspace/bstackdemo/src/test/resources/features/login.feature:32)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- runners.TestRunner -->
